src/components/CartProduct.js

```diff
@@ -0,0 +1,68 @@
+import "../style/cartproduct.css";
```

Comment:
You can read about path alias and use it here

Reply:
okay